src/ZendeskApi_v2/Requests/HelpCenter/Categories.cs

```diff
@@ -30,6 +30,10 @@ public class Categories : Core, ICategories
     {
         private string Locale { get; set; }
 
+        private string CategoriesPath => string.IsNullOrEmpty(Locale)
```

Comment:
What about when the Locale is set to whitespace?
Maybe we should change this to be 
```C# 
string.IsNullOrWhitespace()
```

Reply:
Agree, done!
Thanks :)